### PR TITLE
GitHub assets

### PR DESCRIPTION
**Pull request by William-Hill**
Fixes #

## Proposed Changes

  - Fetch ESGF resources from GitHub Assets
  - Removed hardcoded version numbers
  - Fixed typos

### CODE REVIEW

File: group_vars/data.yml [hunk 1]
```diff
@@ -7,17 +7,32 @@ thredds:
...
   name: thredds
 
+stats_api_version: v1.0.9
+
 stats_webapp:
-  url: "{{ mirror.base }}/2.8/1/esgf-stats-api/esgf-stats-api.war"
+  url: "{{ assets.stats_api }}/esgf-stats-api.war"
   name: esgf-stats-api
 
+dashboard:
```
Comment (nathanlcarlson) [inline group_vars/data.yml]: Can you remove this whole dashboard block here. It conflicts with that in group_vars/all.yml

**Comment by nathanlcarlson** [comment]
It looks like the Java Runtime version used to build the .jars for AccessLogging (and likely other .jars) is relevant. Thredds failed to start the AccessLogging Filter with following message:
```
 java.lang.UnsupportedClassVersionError: esg/node/filters/AccessLoggingFilter has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0 (unable to load class [esg.node.filters.AccessLoggingFilter])
```

**Comment by William-Hill** [comment]
@nathanlcarlson I have java 8 on my laptop, but somehow I have a Java 11 compiler (JRE class number 55).  I'm trying to recompile with a Java 8 compiler (JRE class number 52).

**Comment by William-Hill** [comment]
Closing to create a fresh PR with a cleaner history

**Pull request closed without merging**